bedrock/firefox/templates/firefox/choose.html

```diff
@@ -33,7 +33,13 @@ <h2 data-tracking="{{ _('Browse Freely.<br> Choose Firefox.') }}" data-choice="{
 <section class="page-content">
   <div class="blurb">
     <h2>{{ _('The most private browsing') }}</h2>
-    <p>{{ _('Take control of your web by using Private Browsing with Tracking Protection, only from Firefox.') }}</p>
+    <p>
+      {% if LANG == 'en-US' %}
```

Comment:
We have en-GB, but I don't see anything that requires a change in this sentence, so OK for en-*